packages/@atjson/source-html/test/converter.test.ts

```diff
@@ -518,6 +518,28 @@ describe("@atjson/source-html", () => {
+<script async src="https://www.threads.net/embed.js"></script>
+`
+      ).convertTo(OffsetSource);
+
+      let hir = new HIR(doc).toJSON();
+      expect(hir).toMatchObject({
```

Comment:
Note: when running tests, please use `serialize(doc, { withStableIds: true })` instead of the HIR. The HIR package is deprecated as of now.